api/app.js

```diff
@@ -24,6 +24,7 @@ app.set("view engine", "jade");
 app.use(cors());
 app.use(logger("dev"));
 // data from client side - req.body
+// Express's version of body parser
```

Comment:
That's nice! I need to learn more about the express version of the body parser! :)